charts/matrix-stack/Chart.yaml

```
@@ -4,7 +4,27 @@
 
 apiVersion: v2
 name: matrix-stack
-description: A Helm meta-chart for deploying a Matrix Stack from Element
-type: application
 version: 25.10.3-dev
+kubeVersion: '>= 1.22.0'
+description: A Helm chart for deploying a Matrix Stack from Element
+type: application
+keywords:
+  - ESS
+  - Element
+  - Matrix
+  - Synapse
+  - ElementWeb
+  - MAS
+  - ElementAdmin
+  - MatrixRTC
```

```suggestion
  - MatrixRTC
  - LiveKit
```

Unsure about this one but maybe?